components/ai_chat/content/browser/page_content_fetcher.cc

```diff
@@ -104,6 +128,33 @@ class PageContentFetcher {
         std::move(callback), invalidation_token));
   }
 
+  void StartGithub(
```

Comment:
does this fail if the site (github) throws a certificate error? i would assume so.